assets/src/scss/new-identity/style.scss

```diff
@@ -121,11 +121,18 @@ table.spreadsheet-table.is-color-gp-green {
   }
 }
 
-a:where(.standalone-link) {
+.standalone-link {
   width: fit-content;
-  border-bottom: 2px solid var(--gp-green-500);
   font-weight: 700 !important;
-  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+}
+
+.standalone-link,
+.post-details p > a,
```

Comment:
Actually, I see that for the [Lightbox](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/components/Lightbox/setupLightboxForImages.js#L48) we use the `.post-content` selector rather than `.post-details` so maybe we could do that here too, for consistency?
Also, I think that maybe we can simplify this a bit by writing: 
```
.post-content,
.page-content {
  .standalone-link,
  p > a,
  li > a {
    ...
  }
}
```
What do you think? 🙂

Reply:
Maybe it would also be nice to add a small comment like `// Body link` to clarify what this selector is for 🤔